test_runner/src/test/kotlin/ftl/util/LogTableBuilderTest.kt

```diff
@@ -0,0 +1,112 @@
+package ftl.util
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+
+internal class LogTableBuilderTest {
+
+    private val sampleColumns = arrayOf(
+        TableColumn("header1", "value1", 10),
+        TableColumn("header2", "value2", 15),
+        TableColumn("header3", "value3", 20),
+        TableColumn("header4", "value4", 21)
+    )
+
+    @Test
+    fun `all table lines should have equal size`() {
+        // when
+        val table = buildTable(*sampleColumns)
+
+        // then
+        val tableLines = table.split(System.lineSeparator())
+        tableLines
+            .map { it.length }
+            .zipWithNext()
+            .forEach { (previous, next) ->
+                assert(previous == next)
+            }
+    }
+
+    @Test
+    fun `table first line should contains correct characters`() {
```

Comment:
What do you think of testing the table against a reference output?


```
┌─────────┬────────────────────────┬────────────────────┐
│ OUTCOME │    TEST_AXIS_VALUE     │    TEST_DETAILS    │
├─────────┼────────────────────────┼────────────────────┤
│ Failed  │ walleye-26-en-portrait │ Test failed to run │
└─────────┴────────────────────────┴────────────────────┘
```

This seems easier to understand since we're testing the behavior instead of the implementation details.

Reply:
yes I will add comparing with reference table in UtilsTests :)